app/sms/server/services/voxtelesys.js

```diff
@@ -0,0 +1,78 @@
+import { settings } from '/app/settings';
+import { SMS } from '../SMS';
+const http = require('request');
+
+class Voxtelesys {
+	constructor() {
+		this.authToken = settings.get('SMS_Voxtelesys_authToken');
+	}
+	parse(data) {
+		const returnData = {
+			from: data.from,
+			to: data.to,
+			body: data.body,
+
+			extra: {
+				received_at: data.received_at,
+			},
+		};
+
+		returnData.media = []; /* MMS not currently supported */
+
+		return returnData;
+	}
+	send(fromNumber, toNumber, message) {
+		const options = {
+			timeout: 30000,
+			followRedirect: false,
+			url: 'https://servicelayer01.voxtelesys.net:3100/api/v1/sms',
```

Comment:
Can you maybe make this url configurable in the event they update it in the future?

Reply:
Yes. I can add to the SMS setting.js if this is acceptable? Default will be existing URL.
Something like below:
![sms_configs](https://user-images.githubusercontent.com/36895232/54562591-50704300-4995-11e9-8403-3ec97e8fba8c.png)

Reply:
@graywolf336 changes are below.